src/ui/public/field_format_editor/editors/url/icons/index.js

```diff
@@ -0,0 +1,7 @@
+import '!!file?name=[path][name].[ext]!ui/field_format_editor/editors/url/icons/go.png';
+import '!!file?name=[path][name].[ext]!ui/field_format_editor/editors/url/icons/stop.png';
+import '!!file?name=[path][name].[ext]!ui/field_format_editor/editors/url/icons/de.png';
+import '!!file?name=[path][name].[ext]!ui/field_format_editor/editors/url/icons/ne.png';
+import '!!file?name=[path][name].[ext]!ui/field_format_editor/editors/url/icons/us.png';
+import '!!file?name=[path][name].[ext]!ui/field_format_editor/editors/url/icons/ni.png';
+import '!!file?name=[path][name].[ext]!ui/field_format_editor/editors/url/icons/cv.png';
```

Comment:
This makes me sad. But you're just moving the file, so :shrug: we can solve it later